Doc/library/typing.rst

```diff
@@ -2604,10 +2604,12 @@ Functions and decorators
 
 .. function:: reveal_type(obj, /)
 
-   Reveal the inferred static type of an expression.
+   Ask a static type checker to reveal the statically inferred type of an
+   expression.
```

Comment:
Having "static" twice in the same sentence is awkward, and feels redundant -- if it's a static type checker, then, by definition, it has no other way of inferring the type of an expression other than static inference :)

```suggestion
   Ask a static type checker to reveal the inferred type of an expression.
```